composer.json

```diff
@@ -36,7 +36,8 @@
   "require-dev": {
     "phpunit/phpunit": "4.8.*",
     "squizlabs/php_codesniffer": "2.*",
-    "phpdocumentor/phpdocumentor": "2.8.*"
+    "phpdocumentor/phpdocumentor": "2.8.*",
+    "satooshi/php-coveralls": "dev-master"
```

Comment:
[Documentation](https://github.com/satooshi/php-coveralls#install-by-composer):

> If you need support for PHP versions older than 5.5, you will need to use a 1.x version.

Since the library requires >=5.5, I used dev-master as it said elsewhere. I see you've contributed to the project, so I can definitely change it if 1.0 is the correct way to go.